login.md

```diff
@@ -87,7 +87,7 @@ pwd is /bin/pwd
 
 `pwd` is a shell built-in. Because it is a shell built-in, it will take precedence over external programs. Notice in the the output above it is mentioned first.
```

Comment:
```suggestion
`pwd` is a shell built-in. Because it is a shell built-in, it will take precedence over external programs. Notice in the output above it is mentioned first.
```